sangbok/main.xml

```diff
@@ -4645,6 +4645,29 @@
 			kommer aldrig mer igen. Tjo!
 		</p>
 	</song>
+
+	<song name="Satanvisan"
+				category="Esotorica"
+				melody="Staffan var en stalledräng">
+			<p>
+				En elak man från Helsingfors 
+				Han åkallade Satan
+				Med blåvitt inverterat kors
+				Men han var för lat, han 
+				Stackars lille djävulspräst 
+				Finska Necronomicon är svårläst
+			</p>
+			<p>
+				Gillar du att äta lik?
+				Ond bråd död, ditt levebröd
+				Ja, då är varje kyrkogård ett fik 
+				Ingen tid för griftefrid
+				Offerbål i natten vi tända
+				Kom och låt oss gravarna skända 
+				Varje natt i detta land
+				En kyrka blir satt i brand
+			</p>
+		</song>
```

Comment:
Kan du fixa till indenteringen så att `<p>`-taggarna ligger ett tabbsteg inskjutna, attributen på `<song>` ligger i linje och den sista `</song>` ligger på samma indentering som starttaggen? ;)

I övrigt ser det bra ut, inga problem att lägga in den!